draft-rescorla-tls-esni.md

```diff
@@ -378,21 +411,25 @@ for other purposes within the same message (e.g., encrypting other
 values). Those usages MUST have their own HKDF labels to avoid
 reuse.
 
-[[OPEN ISSUE: If in future you were to reuse these keys for
+[[OPEN ISSUE: If in the future you were to reuse these keys for
```

Comment:
I think either of these is correct.

Reply:
The article seems necessary. Or at least it reads odd without it.